build/NuSpecs/build-nupkg.ps1

```diff
@@ -154,6 +156,8 @@ Write-Host
 
 if(-not $SkipFrameworkPackage)
 {
+    ConfigureNugetCommandLine("Microsoft.ProjectReunion.FrameworkPackage") 
+   
```

Comment:
This should not be a differently named nuget package. It's important that between the prerelease and the release packages the nuget package name stays the same but the implementation changes from dll-based deployment to framework package-based deployment.